client/src/store/language.js

```diff
@@ -4,7 +4,8 @@ const localStorage = window.localStorage.getItem('lang'); // Get last seleceted
 
 const lang = atom({
   key: 'lang',
-  default: localStorage || 'en', // Fall back to English if localStorage is null
+  // If localStorage is null, defaults to Chinese if the domain is iAITok.com; English otherwise
+  default: localStorage || (window.location.host === 'iAITok.com' ? 'cn' : 'en'),
```

Comment:
@BlitzWrecker can we use 'cn' as default for all domains? as we currently mostly chinese users. We might have to use aitok.ai since we can only use one domain (instead of multiple domains) FYI: @jinzishuai

Reply:
I tried that (the first two commits). It is completely okay besides the fact that it will fail Danny's test cases since Danny's tests are written under the assumption that the UI is in English. I can change all tests to account for Chinese UI, but I am not sure if I can finish before Sunday.